ui/components/component-library/text/deprecated/text.js

```diff
@@ -146,10 +146,10 @@ Text.propTypes = {
    */
   textTransform: PropTypes.oneOf(Object.values(TEXT_TRANSFORM)),
   /**
-   * The text-align of the Text component. Should use the TEXT_ALIGN object from
+   * The text-align of the Text component. Should use the TextAlign enum from
    * ./ui/helpers/constants/design-system.js
    */
-  textAlign: PropTypes.oneOf(Object.values(TEXT_ALIGN)),
+  textAlign: PropTypes.oneOf(Object.values(TextAlign)),
   /**
    * Change the dir (direction) global attribute of text to support the direction a language is written
    * Possible values: `LEFT_TO_RIGHT` (default), `RIGHT_TO_LEFT`, `AUTO` (user agent decides)
```

Comment:
From a quick search it looks like this entire folder is only being used once. Maybe we can delete it all together once we update that once instance?

<img width="950" alt="Screenshot 2023-05-20 at 11 36 17 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/d115ee2a-05c9-4af3-a9f6-6ceb5b4996a9">